include/sender_cadmium.hpp

```diff
@@ -106,25 +108,27 @@ class Sender {
 
         // external transition
         void external_transition(
-		                TIME e,
-	                        typename make_message_bags<input_ports>::type mbs) { 
+	    TIME e,
+	        typename
+		    make_message_bags<input_ports>::type mbs) { 
             if((get_messages<typename defs::control_in>(mbs).size()
-				+get_messages<typename defs::ack_in>(mbs).size()) > 1) 
+	        +get_messages<typename defs::ack_in>(mbs).size()) > 1) 
                 assert(false && "one message per time uniti");
-            for(const auto &x : get_messages<typename defs::control_in>(mbs)) {
+            for(const auto &x :
+	        get_messages<typename defs::control_in>(mbs)) {
```

Comment:
It is not required to break this line (it will satisfy < 80 characters).

Reply:
It is exactly 80 characters so it does not meet < 80 characters requirement